Channels/Production/release_notes.md

```diff
@@ -1,27 +1,28 @@
-## April 12, 2021 Production Release ([v1.1.1562.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.1562.01))
+## July 21, 2021 Production Release ([v1.1.1662.02](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.1662.02))
 
-Welcome to the April 12, 2021 Production release of Accessibility Insights for Windows.
+Welcome to the July 21, 2021 Production release of Accessibility Insights for Windows.
 
-Installation Link: https://github.com/microsoft/accessibility-insights-windows/releases/download/v1.1.1562.01/AccessibilityInsights.msi
+Installation Link: https://github.com/microsoft/accessibility-insights-windows/releases/download/v1.1.1662.02/AccessibilityInsights.msi
 
 Documentation Link: https://accessibilityinsights.io/docs/en/windows/overview
 
 ### Highlights
 
+- [New Features](#new-features)
 - [Rule Updates](#rule-updates)
 - [Bug Fixes](#bug-fixes)
 
+#### New Features
+
+- Add support for inspecting and saving [Custom UIAutomation properties](https://docs.microsoft.com/en-us/windows/win32/winauto/uiauto-propertiesoverview). Details on how to configure this feature can be found [here](https://accessibilityinsights.io/docs/en/windows/reference/faq/)
```

Comment:
@codeofdusk, what's the section name for the updated FAQ that documents custom properties?